### ISSUE TITLE
multiple features execution fails

### ISSUE THREAD

**Issue by jlin412** [TheBrainFamily/cypress-cucumber-example#2]
When I execute both facebook.feature and google.feature with cypress open, facebook.feature passes but google.feature fails with url visit with wrong url.  Instead of https//www.google.com I see https://www.google.com/__/#/tests/__all.

When I try recording in dashboard (node_modules/cypress/bin/cypress run --record), it is stuck at the beginning of second feature: google.feature.

When I only execute with one feature, it is working as expected.

**Comment by caznew277bw** [comment]
When I attempt to run the ScenarioOutline.feature I get errors
![screen shot 2018-03-08 at 15 24 14](https://user-images.githubusercontent.com/34125342/37159088-d52bb526-22e4-11e8-8f92-9a5534a7f66b.png)
Could it be missing some dependencies? All the others run ok.

**Comment by lgandecki** [comment]
@jlin412 I tried without using the plugin, just pure JS:

```javascript
it('works', () => {
  cy.visit('https://google.com')
  cy.get('button')
})
```

and it breakes as well - somehow google jumps out of cypress now? Not sure. I don't think this is related to the plugin. Could you run the simple test I pasted above and verify please?

@caznew277bw please open a new issue, this is completely unrelated to @jlin412 problem, and not even related to this repo :). I don't think this is dependency problem though - if Object.entries is not a function it suggests that you are using some old version of node, could you tell us (in a separate issue, here: https://github.com/TheBrainFamily/cypress-cucumber-preprocessor/issues ) what do you get when you run node -v ?

**Comment by jlin412** [comment]
@lgandecki When I revert the order of feature file (aGoogle.feature first), it is working.  Weird!

**Comment by gokulthirunavukarasu** [comment]
I have two feature file one is login and another one is creation, once login feature file is completed before starting the next feature file application getting its logged out
